docs/reference/changelog.md

```diff
@@ -65,24 +67,24 @@ Removed:
 
 #### OC4IDS project schema
 
-* [#127](https://github.com/open-contracting/infrastructure/issues/127) - remove the requirement that linked OCDS releases must be provided in release packages containing only one release. Remove recommendation that OCDS releases are cached from schema and add guidance on caching releases from unreliable sources to implementation guidance
-* [#132](https://github.com/open-contracting/infrastructure/issues/132) - add a publicAuthority organization reference field  
-* [#139](https://github.com/open-contracting/infrastructure/issues/139) - update properties of fields in common with OCDS to version [1.1.4](https://standard.open-contracting.org/1.1/en/schema/changelog/#id1)
+* [#127](https://github.com/open-contracting/infrastructure/issues/127) - remove the requirement that linked OCDS releases must be provided in release packages containing only one release. Remove recommendation that OCDS releases are cached from schema and add guidance on caching releases from unreliable sources to implementation guidance.
+* [#132](https://github.com/open-contracting/infrastructure/issues/132) - add a publicAuthority organization reference field  .
```

Comment:
Remove whitespace before period.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/288/commits/54bc6f2e541f81e412c5aeee58df7798d921e346